app/models/hud_reports/report_instance.rb

```diff
@@ -108,26 +108,25 @@ def related_job
     # FIXME: maybe a single question column on report_instance to track if this is a single
     # question run or all questions.... Need better start/complete logic
     def start(question, tables)
-      universe(question).update(status: 'Started', metadata: { tables: Array(tables) })
-      start_report if build_for_questions.count == remaining_questions.count
```

Comment:
The report is now marked as started when the report job calls `@generator.prepare_report`. This is intended to be a simplification that lets the job's run-time include time spent in prepare_report (building hopwa caper enrollments). I think this is okay assuming we're always running questions via the job.

Reply:
@ttoomey I'm confused, does this mean `start` doesn't call `start_report` ? Where is `start_report` getting called now?

Reply:
@eanders Yes, report.start only starts the question. The report is marked as started before that: RunReportJob calls generator.prepare_report which calls report.start_report. Happy to huddle about this if it would help

Reply:
That's what I was looking for.  Thanks.